connector.go

```diff
@@ -273,33 +272,33 @@ func (c *Connector) generateTopicFromProto(msg proto.Message) kafkautils.Topic {
 }
 
 // RegisterProtos generates kafka topic and protobuf type mappings from proto.Message and registers them dynamically.
-func (c *Connector) RegisterProtos(protos ...proto.Message) {
-	if c.env == kafkautils.Dev {
+func (c *Connector) RegisterProtos(msgType kafkautils.MsgType, protos ...proto.Message) {
+	if c.env == kafkautils.EnvDev {
 		log.Debug().Msg("protoregistry is disabled in dev mode, set kafka.env to other values (e.g., test, staging) to enable it")
 		return
 	}
 
-	tt := c.buildTopicTypes(protos...)
+	tt := c.buildTopicTypes(msgType, protos...)
 
 	err := c.ProtoRegistryCli.RegisterDynamicTopics(tt, c.MsgType)
```

Comment:
the second parameter uses `c.MsgType`, they are not consistent

Reply:
yes that's a bug